ci/set_pyarrow_integration.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Please add `set -xeo pipefail` after the shebang.

Reply:
done